eugenio-cyber/digital-airport
# Add NavigationHeader component

#### Description (What was done? Why it was done? How it was done?)

This PR adds the NavigationHeader component, following the task guidelines and requirements. For this, it was necessary to create and improve the layout, also create the Context Provider.

---

#### References

- Related to task: [Navigation Header]

---

#### Screenshots or GIFs (Optional)

![image](https://github.com/user-attachments/assets/13008563-d26b-49bc-9618-fac03bbdfcaf)

---

#### Checklist

- [X] Optimized the images (if any);
- [ ] Reviewed by at least one team member;

---

#### How to test

1. Checkout the branch: `git checkout feature/navigation-header`
2. Install dependencies: `npm install`
3. Run the application: `npm run dev`
4. Create .env file
5. Create a simple flights page to see NavigationHeader component
6. Navigate to Home page and click in "Informações sobre voos"

---

#### Questions or Notes (Optional)

The back arrow icon is the Notion task, but we can use the Figma.
